pkg/controller/hyperconverged/test-files/upgradePatches/upgradePatches.json

```diff
@@ -0,0 +1 @@
+../../../../../assets/upgradePatches.json
```

Comment:
Is this the desired content of the file?

Reply:
no, this is how github renders a symlink to the real file under `../../../../../assets/upgradePatches.json`